src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -58,8 +58,8 @@ export default class BusinessTypeConfirm extends Vue {
   /** Whether to display Change features. */
   @Prop({ default: false }) readonly hasBusinessTypeChecked!: boolean
 
-  /** Whether the business type is SP or GP. */
-  @Prop({ default: false }) readonly isTypePartnership!: boolean
+  /** Whether entity is a GP (else we assume it's a SP). */
+  @Prop({ default: false }) readonly isEntityPartnership!: boolean
```

Comment:
I'm thinking, why don't we use the getter here directly? The prop's name is the same as the getter and in the parent, we're setting the value as the getter as well. Thoughts Sev?

Reply:
Because this is a sub-component that doesn't really need to know about the store, so it doesn't use getters or actions. This makes it less tied to the architecture (ie, store and getter name). I've found that keeping components as dumb as possible to be better generally. Basically, keeping component scope as small as possible.

Reply:
Sounds good.

Reply:
Another way to look at it is that this component has a well-defined API (its inputs and outputs). It doesn't need to know about external/global stuff.

This generally makes for a cleaner architecture (though Jonathan might disagree). But it does makes testing components easier :)

Reply:
I remember Jonathan used to push against that haha. Makes so much sense to me. Thank you Sev!